src/validation-strategies/in-active-sprint.js

```diff
@@ -0,0 +1,33 @@
+import 'colors';
+
+function withinActiveSprint(issue, jiraClientAPI) {
+  let today = new Date();
+
+  return jiraClientAPI.findIssue(issue)
+    .then(contents => {
+      let fields = contents.fields.customfield_10804;
```

Comment:
Custom field? Do you need to acquire this id from Jira similar to how we are with Epics?